### PR TITLE
LPS-114069 Replace layout classes with new clay components in headless-discovery-web

### PR DESCRIPTION
**Pull request by eduardoallegrini**
This is a simple code replacement, there are a couple of classes we might remove but I'll comment the lines

### CODE REVIEW

**Comment by eduardoallegrini** [comment]
ci:test:sf

**Comment by eduardoallegrini** [comment]
ci:test:stable

**Comment by eduardoallegrini** [comment]
ci:test:relevant

File: modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/LanguageListItemEditable.es.js [hunk 1]
```diff
@@ -160,21 +161,25 @@ const LanguageListItem = ({
 			ref={itemRef}
 		>
 			<ClayTable.Cell expanded>
-				<div className="autofit-row autofit-row-center">
-					<div className="autofit-col autofit-padded-no-gutters">
+				<ClayLayout.ContentRow verticalAlign="center">
+					<ClayLayout.ContentCol className="autofit-padded-no-gutters">
```
[the remark this answers is not in the document]
Comment (boton) [reply modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/LanguageListItemEditable.es.js:165]: > we should keep it as it is for now

👍

**Comment by jbalsas** [comment]
ci:forward

**Comment by nikki-pru** [comment]
For this PR, there's a failure on
`[exec] > Task :apps:segments:segments-web:packageRunTest FAILED`
`[exec] FAIL test/js/components/segment_edit/SegmentEdit.es.js (12.951s)`

[The full log can be found here](https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/343512/consoleText)

**Comment by eduardoallegrini** [comment]
Thanks @nikki-pru, I'm going to take a look at it 🙂

**Comment by dgarciasarai** [comment]
In `master` branch `segments-web` tests are passing successfully:

Screenshot 2020-06-12 at 10 46 25

I saw that this pull contains some changes in `segments-web` module so the testing snapshots changed as well. Take a look at the changes, and if it's aligned to what you changed in the code, regen the snapshots with `yarn test --updateSnapshot`. Be careful please to not broke any test.

Thanks.

**Comment by eduardoallegrini** [comment]
updated here https://github.com/eduardoallegrini/liferay-portal/pull/91

thanks everyone!

**Pull request closed without merging**